README.md

```diff
@@ -1,40 +1,57 @@
-# USE https://github.com/Financial-Times/upp-global-configs FOR GLOBAL CONFIGS
-
 ## Description
-This repository contains the files required to provision clusters using Kubernetes on AWS for UPP (delivery, publishing) and PAC platforms.
 
-It uses [kube-aws](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) for provisioning the cluster on AWS.
+This repository contains core automation required to create/update/destroy Kubernetes clusters for UPP (delivery, publishing) and PAC platforms. The setup uses the [kube-aws](https://github.com/kubernetes-incubator/kube-aws) tool to manage Kubernetes infrastructure on AWS. Ansible and bash are used to script and integrate kube-aws with any additional tasks and Docker to package the provisioner setup.
+
+
+* [Prerequisites](#prerequisites)
+* [Build the provisioner Docker image](#build-the-provisioner-docker-image)
+* [Provision a new cluster](#provision-a-new-cluster)
+* [Update a cluster](#update-a-cluster)
+* [Decommission a cluster](#decommission-a-cluster)
+* [Access a cluster](#access-a-cluster)
+* [Restore Kubernetes Configuration](#restore-kubernetes-configuration)
+* [Rotate TLS assets of a cluster](#rotate-tls-assets-of-a-cluster)
+  * [Troubleshooting](#troubleshooting)
+    * [After rotation one could not login using the normal flow](#after-rotation-one-could-not-login-using-the-normal-flow)
+* [Upgrade kube-aws](#upgrade-kube-aws)
+  * [Validate that the update works](#validate-that-the-update-works)
+
 
 ## Prerequisites
-1. [Install docker](https://docs.docker.com/engine/installation/) locally
-2. [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl), the latest version (preferably >1.8). OSX users can install using brew.
-```
-brew install kubernetes-cli
-```
-You can check the client version by running 
-```
-kubectl version
-```
-3. [Install helm](https://github.com/kubernetes/helm#install), the latest version (preferably >2.7).
 
-## Building the Docker image
-The k8s provisioner can be built locally as a Docker image:
 
-```
+To use the provisioner the below tools need to be available:
+
+- [docker](https://docs.docker.com/engine/installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+- [helm](https://github.com/kubernetes/helm#install)
```

Comment:
Actually we are using this reference for helm installation because of incompatibility between our version and latest one:

https://confluence.ft.com/pages/viewpage.action?spaceKey=CONTENT&title=Getting+Onboard

> nstall Helm. Our server Helm version is 2.9.1 while the latest client version is 2.12.2 and they are not compatible, so:
If you have Helm already installed "brew unlink kubernetes-helm"
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/78d64252f30a12b6f4b3ce29686ab5e262eea812/Formula/kubernetes-helm.rb

brew pin kubernetes-helm (If you want to upgrade not pinned packages and not get error exit code from brew use - brew upgrade --ignore-pinned)

Reply:
I added more details about the current version we use. @mihaylovmihail please review it again.